frontend/src/api/k8s/templates.ts

```diff
@@ -15,3 +15,25 @@ export const listTemplates = (
     queryOptions,
   }).then((listResource) => listResource.items);
 };
+
+export const toggleTemplateEnabledStatus = (
+  name: string,
+  namespace: string,
+  enable: boolean,
+): Promise<TemplateKind> => {
+  const patch = enable
+    ? {
+        op: 'remove',
+        path: '/metadata/annotations/opendatahub.io~1template-enabled',
+      }
+    : {
+        op: 'add',
+        path: '/metadata/annotations/opendatahub.io~1template-enabled',
+        value: 'false',
```

Comment:
```suggestion
        op: 'replace',
        path: '/metadata/annotations/opendatahub.io~1template-enabled',
        value: 'true',
      }
    : {
        op: 'replace',
        path: '/metadata/annotations/opendatahub.io~1template-enabled',
        value: 'false',
```

Nit, I'm not against this flow, but seems weird that the label is either there or not with a boolean, why not have it in all cases, as you then have the checks against `temaplate-enabled=false` the logic won't change and it's more obvious checking the yaml.

Reply:
Yeah, this is a good idea. So only when the annotation is `false` do we disable the template, otherwise (`true` or non-exist) we think the template is enabled.